communities/rpc/src/lib.rs

```diff
@@ -44,26 +50,40 @@ pub trait CommunitiesApi<BlockHash> {
 		cid: CommunityIdentifier,
 		at: Option<BlockHash>,
 	) -> Result<Vec<Location>>;
+
+	#[rpc(name = "communities_getAllBalances")]
+	fn communities_get_all_balances(
```

Comment:
Haha @brenzi, we could get rid of the irregularities here. ;)

Reply:
yeah, let's prefix all our rpcs with `encointer`, not with the pallet name

Reply:
so, `encointer_getAllBalances`

Reply:
I agree, with `encointer_getAllBalances`